src/test/java/org/zalando/problem/spring/web/advice/AdviceTraitTest.java

```diff
@@ -81,6 +82,17 @@ public void buildsOnThrowable() throws HttpMediaTypeNotAcceptableException {
     }
 
     @Test
+    public void buildsOnThrowableUsingHttpStatus() throws HttpMediaTypeNotAcceptableException {
+        final ResponseEntity<Problem> result = unit.create(HttpStatus.RESET_CONTENT,
+                                                           new IllegalStateException("Message"), request());
```

Comment:
Can you change your formatting style for continuing method invocations to 8 spaces (no alignment on parameters)?